snuba/datasets/configuration/generic_metrics/storages/sets.yaml

```diff
@@ -71,5 +71,11 @@ allocation_policy:
       org_limit_bytes_scanned: 100000
 
 query_processors:
+  - processor: MappingOptimizer
+    args:
+      column_name: tags
+      hash_map_name: _raw_tags_hash
+      killswitch: generic_metrics/tags_hash_map_enabled
+      value_subcolumn_name: raw_value
```

Comment:
Same comment about this column not being available yet. We can remove this for now